src/rprblender/engine/viewport_engine.py

```diff
@@ -515,6 +515,7 @@ def sync(self, context, depsgraph):
         self.setup_upscale_filter({
             'enable': scene.rpr.viewport_upscale,
             'resolution': (self.width, self.height),
+            'scene': scene,
```

Comment:
no need this

Reply:
fixed https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/pull/577/commits/bdb00ef37826dc9b26876222f8cd5e316a372e6b